service/resources/arcgis.py

```diff
@@ -0,0 +1,103 @@
+"""ArcGIS module"""
+import os
+import math
+import json
+import requests
+import falcon
+import jsend
+import usaddress
+
+class Arcgis():
+    """Welcome class"""
+    def on_get(self, req, resp, name):
+        """on get request
+        return Arcgis message
+        """
+        if name in dir(self):
+            return getattr(self, name)(req, resp)
+        msg = {'message': 'Welcome'}
+        resp.body = json.dumps(jsend.success(msg))
+        resp.status = falcon.HTTP_200
+
+    def parcels(self, req, resp):
+        """ get parcels """
+
+        missing_env = self.has_missing_env(['PLN_ARCGIS_PARCEL'])
+        if missing_env:
+            resp.status = falcon.HTTP_404
+            msg_error = jsend.error(
+                'Missing or invalid environment variable(s): '+", ".join(missing_env))
+            resp.body = json.dumps(msg_error)
+            return
+
+        response = {'message': 'parcels'}
+        if 'address' in req.params:
+            addr = usaddress.tag(req.params['address'])
+            options = {'returnGeometry':'false', 'returnSuggestions':False}
```

Comment:
Kind of weird that one is string and the other is boolean

Reply:
commit 961212fb64fb4f346b8cfc94e2fab0cf577fa1a0